html/captive-portal/lib/captiveportal/DynamicRouting/Module/SelectRole.pm

```diff
@@ -0,0 +1,47 @@
+package captiveportal::DynamicRouting::Module::SelectRole;
+use Moose;
+
+BEGIN { extends 'captiveportal::PacketFence::DynamicRouting::Module::SelectRole'; }
+
+=head1 NAME
+
+captiveportal::DynamicRouting::Module::SelectRole - SelectRole Controller for captiveportal
+
+=head1 DESCRIPTION
+
+[enter your description here]
```

Comment:
Module to select a new role for the device being registered

Reply:
Given its the «extension» point, we usually don't put a description there
At least, thats how the others are, IDK if it was the way it was meant to be but I find it to be a good idea